build.sbt

```diff
@@ -423,6 +423,12 @@ val mimaFilterSettings = Seq {
     // Documentation for scala.jdk package
     ProblemFilters.exclude[MissingClassProblem]("scala.jdk.package"),
     ProblemFilters.exclude[MissingClassProblem]("scala.jdk.package$"),
+
+    // Convenience method for copying collections to Arrays (scala/scala#9232)
```

Comment:
The useful thing to note is why this method can exist (because it's `private[collection]`), not so much why it's useful (which sounds worst than I intend..)

Reply:
done